pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -426,7 +439,7 @@ func (gce *GCECloud) DeleteExternalTargetPoolAndChecks(name, region, clusterID s
 // verifyUserRequestedIP checks the user-provided IP  to see whether it can be
 // used for the LB.  It also returns whether the IP is considered owned by the
```

Comment:
Can you clarify this comment as the function can now return (true, err != nil), seems confusing.

Reply:
Good point. I changed the function to never return (true, non-nil-err). If the err is non-nil, the boolean value should not be used anyway.